buildSrc/src/main/java/dependencies/Dep.kt

```diff
@@ -46,6 +46,7 @@ object Dep {
         val coreKtx = "androidx.core:core-ktx:1.0.0-alpha1"
         val preference = "androidx.preference:preference:1.0.0"
         val browser = "androidx.browser:browser:1.0.0"
+        val fragment =  "androidx.fragment:fragment:1.1.0-alpha03"
```

Comment:
I think, there seems to be extra space 👀 

```suggestion
        val fragment = "androidx.fragment:fragment:1.1.0-alpha03"
```